.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,3 +2,19 @@
 **Asana Ticket:** [TICKET_NAME](TICKET_LINK)
 
 [Please include a brief description of what was changed]
+
+---
+
+Before getting review, please check the following:
+
+* [ ] Does frontend functionality render and work correctly in IE?
+* [ ] Has every endpoint that will be polled from the frontend been load-tested?
+* [ ] Do form elements have ARIA labels?
+* [ ] Have you checked for tech debt you can address in the area you're working in?
+* [ ] Have you considered the following cases that often require special handling:
```

Comment:
All these items seem to fall under the category of "special _routes_" specifically, so maybe we could condense this to:

```
* [ ] If this change involves routes, does it work correctly with "unusual"
      routes such as the combined Green Line, Silver Line, Foxboro commuter
      rail, and single-direction bus routes like the 170?
```

Reply:
@digitalcora sounds good, I will also add a note that this list is not meant to be exhaustive (and I imagine we'll remember/discover a couple of other weird routes from time to time).